src/lib/list/list-item/list-item-foundation.ts

```diff
@@ -69,7 +69,10 @@ export class ListItemFoundation implements IListItemFoundation {
       this._adapter.setTwoLine(this._twoLine);
     }
 
-    this._adapter.trySelect(this._value);
+    const isSelected = this._adapter.trySelect(this._value);
+    if(isSelected != null) {
```

Comment:
I added this because it's possible for the `_selected` field would be out of sync as well after initialization as well as in the `value` setter.